straxen/plugins/event_processing.py

```diff
@@ -560,45 +565,53 @@ def setup(self):
 
         else:
             raise NotImplementedError('FDC map format not understood.')
-
+            
     def compute(self, events):
 
         result = {'time': events['time'],
                   'endtime': strax.endtime(events)}
         
-        z_obs = - self.electron_drift_velocity * (events['drift_time'] - self.electron_drift_time_gate)
-        orig_pos = np.vstack([events[f's2_x'], events[f's2_y'], z_obs]).T
-        r_obs = np.linalg.norm(orig_pos[:, :2], axis=1)
-        delta_r = self.map(orig_pos)
-
-        # apply radial correction
-        with np.errstate(invalid='ignore', divide='ignore'):
-            r_cor = r_obs + delta_r
-            scale = r_cor / r_obs
-
-        # z correction due to longer drift time for distortion
-        # (geometrical reasoning not valid if |delta_r| > |z_obs|,
-        #  as cathetus cannot be longer than hypothenuse)
-        with np.errstate(invalid='ignore'):
-            z_cor = -(z_obs ** 2 - delta_r ** 2) ** 0.5
-            invalid = np.abs(z_obs) < np.abs(delta_r)
-            # do not apply z correction above gate
-            invalid |= z_obs >= 0
-        z_cor[invalid] = z_obs[invalid]
-        delta_z = z_cor - z_obs
-
-        result.update({'x': orig_pos[:, 0] * scale,
-                       'y': orig_pos[:, 1] * scale,
-                       'r': r_cor,
-                       'r_naive': r_obs,
-                       'r_field_distortion_correction': delta_r,
-                       'theta': np.arctan2(orig_pos[:, 1], orig_pos[:, 0]),
-                       'z_naive': z_obs,
-                       # using z_obs in agreement with the dtype description
-                       # the FDC for z (z_cor) is found to be not reliable (see #527)
-                       'z': z_obs,
-                       'z_field_distortion_correction': delta_z
-                       })
+        for s_i in [0, 1, 2]:
```

Comment:
I was confused for a bit, maybe we could make this a bit more explicit by doing something like 
```python
for field, xy_pos in zip(
   ['drift_time', 'alt_s1_interaction_drift_time', 'alt_s2_interaction_drift_time'],
   ['s2_', 'alt_s2_', 'alt_s2_']
):
or something similar. Otherwise, it certainty would help if `s_i` -> `alt_s_i`  there would be a comment that `alt_s_i == 0` is the main pair.

Reply:
yeah it would be only that, 👍